near/tests/stake_nodes.rs

```diff
@@ -380,3 +380,63 @@ fn test_validator_join() {
         system.run().unwrap();
     });
 }
+
+#[test]
+fn test_inflation() {
```

Comment:
hm, do you need nodes for this? or you can do this on runtime level?

Reply:
I want to check that this works on the block level but runtime doesn't know about blocks